### PR TITLE
WIP: make tests pass

### PR DESCRIPTION
**Pull request by daschw**

### CODE REVIEW

**Comment by daschw** [comment]
This needs https://github.com/JuliaLang/METADATA.jl/pull/17217

File: test/runtests.jl [hunk 1]
```diff
@@ -1,27 +1,30 @@
 module PlotsTests
 
+using Pkg
+Pkg.add(PackageSpec(url="https://github.com/tkf/PlotReferenceImages.jl"))
```
Comment (tkf) [inline test/runtests.jl]: Surely you don't want to use my fork :)

Comment (daschw) [reply test/runtests.jl]: Oh, thanks for taking such a close look!

**Comment by mkborregaard** [comment]
How is this related to #1689? They look partly overlapping

**Comment by mkborregaard** [comment]
This still error on Travis with
```
Building Gtk ─────────────→ `~/.julia/packages/Gtk/oHM1K/deps/build.log`
┌ Error: Error building `Gtk`: 
│ ERROR: LoadError: syntax: misplaced assignment statement in "[glib = library_dependency("glib", aliases = ["libglib-2.0", "libglib-2.0-0"], group = group); gobject = library_dependency("gobject", aliases = ["libgobject-2.0", "libgobject-2.0-0"], group = group); gtk = library_dependency("gtk", aliases = ["libgtk-3", "libgtk-3-0"], group = group); gdk = library_dependency("gdk", aliases = ["libgdk-3", "libgdk-3-0"], group = group); gdk_pixbuf = library_dependency("gdk_pixbuf", aliases = ["libgdk_pixbuf-2.0", "libgdk_pixbuf-2.0-0"], group = group); gio = library_dependency("gio", aliases = ["libgio-2.0", "libgio-2.0-0"], group = group)]"
│ Stacktrace:
│  [1] include at ./boot.jl:317 [inlined]
│  [2] include_relative(::Module, ::String) at ./loading.jl:1038
│  [3] include(::Module, ::String) at ./sysimg.jl:29
│  [4] include(::String) at ./client.jl:388
│  [5] top-level scope at none:0
│ in expression starting at /home/travis/.julia/packages/Gtk/oHM1K/deps/build.jl:7
```

**Comment by daschw** [comment]
See #1700 instead.

**Pull request closed without merging**